wavefront/server/packages/flo_cloud/flo_cloud/azure/synapse.py

```
@@ -0,0 +1,265 @@
+                        f'Query: {converted_query!r}'
+                    )
+                return [dict(zip(columns, row)) for row in cursor.fetchall()]
+            except pyodbc.Error as e:
+                logger.error(f'Synapse query execution error: {e}')
+                raise
+
+    def execute_query_to_dict(
+        self,
+        projection: str = '*',
+        table_prefix: str = '',
+        table_names: List[str] = [],
```

Ruff (0.15.7)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)